app/index.html

```diff
@@ -26,17 +26,21 @@ <h1 class="hero-title">Polymer Project</h1>
   <style>
     .product-grid-item {
       display: inline-block;
-      width: calc(50% - 50px);
-      margin-bottom: 20px;
+      width: calc(50% - 60px);
+      margin-bottom: 40px;
       padding: 20px;
       background: rgba(0,0,0,0.6);
       background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, rgba(0,0,0,0.4) 0%, rgba(0,0,0,0.6) 100%);
-      color: #000;
+      color: rgba(0,0,0,.87);
```

Comment:
Note: alpha text color here is fine - limited use and required for effect with gradient background.